spec/support/spec_config.rb

```diff
@@ -92,6 +92,23 @@ def connect_options
     end
   end
 
+  def supports_scram_256?
+    return @supports_scram_256 if @supports_scram_256
+
+    client = Mongo::Client.new(addresses, Mongo::Options::Redacted.new(
```

Comment:
When this is moved to cluster config, this should use one of the global clients (`ClientRegistry.instance.global_client`) unless there's a reason why this won't work.